set.mm

```diff
@@ -419279,6 +419279,16 @@ the theorem forms ("i", "d" or "g") should be the first of multiple
        </li>
        </ul>
 
+       <li><b>Default form.</b>
```

Comment:
I would propose not to create a new list item for this, but to append this paragraph at the end of list item "Theorem forms.", which ends with

> When an inference is converted to a theorem by eliminating an "is a set"
>  hypothesis, we sometimes suffix the closed form with "g" (for "more
>  general") as in ~ uniex vs. ~ uniexg .  In this case, the inference
>  often has no suffix "i".

Reply:
@avekens this way? https://github.com/metamath/set.mm/pull/2884/commits/2173e350a8fd5fbc96b2cc1d28efab6fe097e7e8